### PR TITLE
Move config.json to root folder

### PR DESCRIPTION
**Pull request by erikzhang**
Replace #707

### CODE REVIEW

**Comment by erikzhang** [comment]
@Liaojinghui Please check it.

File: src/Directory.Build.props [hunk 1]
```diff
@@ -13,6 +13,13 @@
     <RepositoryUrl>https://github.com/neo-project/neo-modules.git</RepositoryUrl>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Update="config.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+    </None>
+  </ItemGroup>
```
Comment (Liaojinghui) [inline src/Directory.Build.props:21]:
```suggestion
    <ItemGroup>
    <Content Include="config.json" Condition="Exists('$(ProjectDir)config.json')">
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
      <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
    </Content>
  </ItemGroup>
```

**Review by Liaojinghui — APPROVED** [review]

**Comment by Liaojinghui** [comment]
Works well on my MacOS.

**Comment by Liaojinghui** [comment]
Works well on my Windows 11

**Comment by cloud8little** [comment]
> Works well on my MacOS.

what is the behavior currently?

**Comment by Liaojinghui** [comment]
@cloud8little now config.json appears under the output directory together with the plugin.dll files

**Comment by cloud8little** [comment]
> @cloud8little now config.json appears under the output directory together with the plugin.dll files

every plugin has a config.json file, are we going to combile them all in one config.json file or put all dll and config file for each plugin under each plugin name folder.  such as:
RpcServer/RpcServer.dll 
RpcServer/config.json
ApplicationLogs/ApplicationLogs.dll
ApplicationLogs/config.json

**Comment by Liaojinghui** [comment]
@cloud8little 

RpcServer/RpcServer.dll
RpcServer/config.json
ApplicationLogs/ApplicationLogs.dll
ApplicationLogs/config.json

**Comment by vncoelho** [comment]
I am getting this error now.

```cs
  neo -> /opt/neo-modules/neoLib/src/neo/bin/Release/net6.0/Neo.dll
  LevelDBStore -> /opt/neo-modules/src/LevelDBStore/bin/Release/net6.0/LevelDBStore.dll
  RpcServer -> /opt/neo-modules/src/RpcServer/bin/Release/net6.0/RpcServer.dll
  ApplicationLogs -> /opt/neo-modules/src/ApplicationLogs/bin/Release/net6.0/ApplicationLogs.dll
/usr/share/dotnet/sdk/6.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.ConflictResolution.targets(112,5): error NETSDK1152: Found multiple publish output files with the same relative path: /opt/neo-modules/src/RpcServer/config.json, /opt/neo-modules/src/ApplicationLogs/config.json. [/opt/neo-modules/src/ApplicationLogs/ApplicationLogs.csproj]
File does not exist
```

**Pull request merged**